InstrumentationEngine.sln

```diff
@@ -588,6 +592,54 @@ Global
+		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.CodeAnalysis.Native|x64.ActiveCfg = Debug|Any CPU
+		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.CodeAnalysis.Native|x64.Build.0 = Debug|Any CPU
+		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.CodeAnalysis.Native|x86.ActiveCfg = Debug|Any CPU
+		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.CodeAnalysis.Native|x86.Build.0 = Debug|Any CPU
+		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Debug|x64.Build.0 = Debug|Any CPU
+		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Debug|x86.Build.0 = Debug|Any CPU
+		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Release|Any CPU.Build.0 = Release|Any CPU
+		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Release|x64.ActiveCfg = Release|Any CPU
+		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Release|x64.Build.0 = Release|Any CPU
+		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Release|x86.ActiveCfg = Release|Any CPU
+		{182D00C1-705F-4FF6-8620-8EEF29936C7A}.Release|x86.Build.0 = Release|Any CPU
+		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Managed|Any CPU.ActiveCfg = Debug|Any CPU
+		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Managed|Any CPU.Build.0 = Debug|Any CPU
+		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Managed|x64.ActiveCfg = Debug|Any CPU
+		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Managed|x64.Build.0 = Debug|Any CPU
+		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Managed|x86.ActiveCfg = Debug|Any CPU
+		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Managed|x86.Build.0 = Debug|Any CPU
+		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Native|Any CPU.ActiveCfg = Debug|Any CPU
+		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Native|Any CPU.Build.0 = Debug|Any CPU
+		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Native|x64.ActiveCfg = Debug|Any CPU
+		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Native|x64.Build.0 = Debug|Any CPU
+		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Native|x86.ActiveCfg = Debug|Any CPU
+		{F356AF58-F432-415D-BE13-44228F265CEF}.CodeAnalysis.Native|x86.Build.0 = Debug|Any CPU
+		{F356AF58-F432-415D-BE13-44228F265CEF}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{F356AF58-F432-415D-BE13-44228F265CEF}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{F356AF58-F432-415D-BE13-44228F265CEF}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{F356AF58-F432-415D-BE13-44228F265CEF}.Debug|x64.Build.0 = Debug|Any CPU
+		{F356AF58-F432-415D-BE13-44228F265CEF}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{F356AF58-F432-415D-BE13-44228F265CEF}.Debug|x86.Build.0 = Debug|Any CPU
```

Comment:
Do these need to be built as part of the platform specific builds?

Reply:
The other projects that rely on the RemoteUnitTestExecutor already build for platform specific builds, so I wanted to keep these the same. In the hopefully near future, I'd like to make all of the managed projects be platform agnostic, which will simplify restoring and building.